StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSavedPaymentMethodsCollectionViewController.swift

```diff
@@ -537,6 +539,7 @@ extension CustomerSavedPaymentMethodsCollectionViewController: UpdatePaymentMeth
             collectionView.reloadData()
             return .success(())
         } catch {
+            // TODO: Implement logic to decide if we should present cardBrandError or generic error
```

Comment:
Nit: Do we have a ticket for this? I could see it getting forgotten kinda easily since it's just error handling stuff.

Reply:
https://jira.corp.stripe.com/browse/MOBILESDK-3300